src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors color = Colors.values()[new Random().nextInt(Colors.values().length)];
+        return color.toString();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/f0a7d510-380e-4194-9097-8d081373d7f2)
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Make random a class-level field instead, to avoid needless recreation per each method execution